### ISSUE TITLE
CDC COVID-19 - Bug: Language attributes are missing for the languages in the bottom of the page

### ISSUE THREAD

**Issue by lamarjordanux** [a11ySea/testing#9]
## Issue Summary

Screen readers may not announce the language properly without lang attributes.

## Steps to reproduce

1. Go to one of the language buttons in the bottom of the page
2. Inspect the button code 

### Expected behavior

Screen reader pronounces the language in native tongue.

### Actual behavior

- Screen reader pronounces language in English 

### Device Information:

 - OS: NVDA
 - Browser Chrome

### Screen Reader or AT Information

 - [e.g. JAWS 2020.2003.13]
 - [e.g. NVDA 2019.3.1]

### Current Code
```
(code snippet)
```

### Suggested Code
```
(code snippet)
```

# Specifications

- **WCAG Principal:**
- **WCAG SC:**
- **Severity:**

## Additional References

- Link to any articles or specs that give further guidance
